Makefile

```diff
@@ -121,11 +121,11 @@ yamllint:
 	@files=$$(find testdata -name '*.yaml' ! -path 'testdata/*/dist/*'); \
     	docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest $$files -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
 
-GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
+TOOLS_BIN_DIR ?= $(shell pwd)/bin
+GOLANGCI_LINT = $(TOOLS_BIN_DIR)/golangci-lint
```

Comment:
Could we please keep it is before and just address the required change to do the fix without any enhancement?

Reply:
Fixed